lib/src/base-styles/_base.scss

```diff
@@ -20,6 +20,7 @@
 @import '../scss-shared/nx-container-helpers';
 @import '../scss-shared/nx-text-helpers';
 @import '../scss-shared/nx-flexbox-helpers';
+@import '../scss-shared/nx-mixins';
```

Comment:
All of our other mixins are in more specific files.  I'd think we'd rather continue with that pattern than make a generic file for mixins.

Reply:
> All of our other mixins are in more specific files. I'd think we'd rather continue with that pattern than make a generic file for mixins.

It didn't seem to fit in any of the other existing scss files (the closest IMO was `container-helpers`) so rather than create a very specific `_mask.scss` I created a generic one in case we end up with other hard to place mixins. I am open to suggestions though.

Reply:
> I created a generic one in case we end up with other hard to place mixins. I am open to suggestions though.

I'd rather stick to our existing pattern which is to have pretty specific mixin files.  I'd call this one `_nx-mask-helpers.scss` to fit the pattern.

Reply:
> > I created a generic one in case we end up with other hard to place mixins. I am open to suggestions though.
> 
> I'd rather stick to our existing pattern which is to have pretty specific mixin files. I'd call this one `_nx-mask-helpers.scss` to fit the pattern.

ok: https://github.com/sonatype/sonatype-react-shared-components/pull/330/commits/3b3e39459dc735a871ec7543b3e066540d3f8891